kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -0,0 +1,588 @@
+//    |  /           |
```

Comment:
how do you plan checking if the element to be used is consistent with this scheme? 

for example that the stiffness matrix and the RHS is ordered correctly...

Reply:
Thinking on that, good point